mphys/multipoint.py

```diff
@@ -69,7 +69,11 @@ def mphys_connect_scenario_coordinate_source(self, source, scenarios, discipline
             for discipline in disciplines_list:
                 src = f'{source}.x_{discipline}0'
                 target = f'{scenario}.x_{discipline}0'
+                print(f'Connecting {src} to {target}')
                 self.connect(src, target)
+        
+        import pdb
```

Comment:
Is this for debugging? Maybe we want to remove it.

Reply:
good catch.